README.md

```diff
@@ -1,22 +1,19 @@
 homebrew-font
 =============
 
-# Install fonts
+# Installing Ricty fonts
 
-Compile Ricty Font
-
-* Install
-   * Recommend
+Recommended installation by tapping the repository
 
 ```
-$ brew tap sanemat/font
+$ brew tap sanemat/homebrew-font
```

Comment:
Oh, I didn't realize it would strip `homebrew-`. I guess either one works, so you can leave out that part of the change if you want.